tests/v2/integration/catalogv2/rancher_managed_charts_test.go

```diff
@@ -125,6 +126,29 @@ func TestRancherManagedChartsSuite(t *testing.T) {
 	suite.Run(t, new(RancherManagedChartsTest))
 }
 
+func (w *RancherManagedChartsTest) TestServeIcons() {
```

Comment:
Can you also add another test where we set the `usesystembundled` setting to `bundled ?

Reply:
That is not possible in the current configuration.
This function:
```
func isRancherAndBundledCatalog(repo repoDef) bool {
	gitDir := git.RepoDir(repo.metadata.Namespace, repo.metadata.Name, repo.status.URL)
	return (git.IsBundled(gitDir) && settings.SystemCatalog.Get() == "bundled")
}
```
Depends on the repository directory for checking for a bundled chart.
It is configured for the following repository paths:
```
const (
	stateDir  = "management-state/git-repo"
	staticDir = "/var/lib/rancher-data/local-catalogs/v2"
	localDir  = "../rancher-data/local-catalogs/v2" // identical to helm.InternalCatalog
)
```
However in the `integration tests environment` it will always return the following directory:
```
❯ docker exec -it d30b65146e1a bash -c "find / -type d -name 'management-state' 2>/dev/null"                                                                                            rancher/git/icons-airgap !
/go/src/github.com/rancher/rancher/build/testdata/management-state
```

Reply:
Added, @rohitsakala